MapboxCoreNavigation/RouteProgress.swift

```diff
@@ -140,6 +140,8 @@ open class RouteProgress: NSObject {
             congestionTravelTimesSegmentsByStep.append(congestionTravelTimesSegmentsByLeg)
         }
     }
+    
+    public var spokenInstructionsForRoute: [String: Data] = [:]
```

Comment:
Can we move this to the PollyVoiceController?

Reply:
Yeah I guess it doesn’t need to be stored on routeprogress